beta/nncf/tensorflow/callbacks/checkpoint_callback.py

```diff
@@ -0,0 +1,70 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from typing import Union
+
+import tensorflow as tf
+
+
+class CheckpointManagerCallback(tf.keras.callbacks.Callback):
```

Comment:
Could you add a test for the CheckpointManagerCallback?

Reply:
Please see https://github.com/openvinotoolkit/nncf/blob/d2c8dd79f04156ed8c50b99998394322b466a202/beta/tests/tensorflow/test_callbacks.py